.github/actions/setup-job/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Setup Job
+description: Common setup for all jobs
+runs:
+    using: 'composite'
+    steps:
+        - name: Setup Node 20
+          uses: actions/setup-node@v4
+          with:
+              node-version: '20'
+              cache: 'yarn'
+              registry-url: 'https://registry.npmjs.org'
+
+        - name: Install dependencies
+          shell: bash
```

Comment:
This is redundant!

Reply:
@Rajdeepc which is redundant? the registry URL